rir/src/compiler/util/safe_builtins_list.cpp

```diff
@@ -5,17 +5,277 @@
 namespace rir {
 namespace pir {
 
-bool SafeBuiltinsList::contains(SEXP builtin) {
+bool SafeBuiltinsList::always(int builtin) {
     static int safeBuiltins[] = {
         findBuiltin("vector"),
-        findBuiltin("c"),
```

Comment:
Where did `c` go?

Reply:
thanks that was an accident. It should be in the semi-safe list